scripts/sd-run-bench.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+NPEXECS=30  # Number of process executions (30 in the warmup paper).
+CSV_FILES=sd-v8-results  # File stem only.
+PDF=sd-v8-plots.pdf
+
+# Generate one CSV file for each process execution.
+i=0
+while [ "$i" -lt "$NPEXECS" ]
+do
+    echo Process execution: "$i"
+    tools/sd_run_perf.py --arch x64 --binary-override-path out.gn/x64.release/d8 benchmarks/sd-v8.json --csv-test-results "$CSV_FILES"-"$i".csv
+    i=`expr $i + 1 `
+done
+
+tools/sd_merge_csv.py --files "$CSV_FILES"
```

Comment:
I guess this tool was merged in another PR?

Reply:
oooh, no. I pushed it to our branch in v8, where our other work that shadows the V8 code lives:

https://github.com/softdevteam/v8/tree/new-sd-runner

So, we need a PR that merges `v8/new-sd-runner` into `sd-master`, I think.

Reply:
I think so. Should it be copied over like the other scripts? Then I think we are free to change v8 version without having to constantly merge things into different branches. If you see what I mean.

In other words. Should we need a v8 branch at all?

Reply:
OK, let me just have a go at that...

Reply:
Another random thought. Do we even need to copy? Can we run them direct from sd-fetch-v8 whilst we are under the `v8` dir? I imagine some of the V8 stuff expects things in certain places, right?

Reply:
Not sure I follow you here?

Reply:
So for example, we currently copy `sd-run-bench.sh` into `v8`. But can we do `../scripts/sd-run-bench.sh` to avoid copying? Same with `sd-build.sh`?

What about `sd-base.js`, `sd-v8.json` and `sd-run.js`? I suppose they need to live in `v8/benchmarks`, and so do indeed need copying?

Reply:
If this sounds like a terrible idea, tell me so, btw :)

Reply:
Ah., I expect so. But I think the other stuff needs to go in `v8/`. I'll push something and you can see what I mean...